config/default/config.json

```diff
@@ -5,6 +5,5 @@
     "USER_ID": "",
     "GIVEN_NAME": "DEFAULT_NAME",
     "SURNAME": "DEFAULT_SURNAME",
-    "EMAIL": "DEFAULT@EMAIL.COM",
-    "SERVER_HOST": "localhost:2992"
```

Comment:
Even without this, will the webpack server be accessible from the network? On a mobile device for example? We used to update that value to the local server IP to make it accessible.